src/Sets/SimpleSparsePolynomialZonotope.jl

```diff
@@ -152,3 +152,45 @@ The set resulting from applying the linear map `M` to `P`.
 function linear_map(M::AbstractMatrix, P::SSPZ)
     return SimpleSparsePolynomialZonotope(M * center(P), M * genmat(P), expmat(P))
 end
+
+"""
+    quadratic_map(Q::Vector{MT}, S::SimpleSparsePolynomialZonotope) where {N, MT<:AbstractMatrix{N}}
+
+Return an overapproximation of the quadratic map of the given zonotope.
```

Comment:
If this is not the exact quadratic map, we should change the implementation. The usual pattern here is to create a lazy set operation `QuadraticMap` and define a method
`overapproximate(Q::QuadraticMap{N, <:SSPZ}, Type{<:SSPZ}) where N`

Reply:
Actually we broke this rule ourselves with the existing `quadratic_map` method. We can keep this for now but I suggest we change it soon again.

Reply:
I'm not sure if it breaks the exact quadratic map or an overapproximation of it

Reply:
I think it's exact, because polynomial zonotopes are supposed to be closed (with formula) under quadratic map (see slide 6 of Niklas presentation

Reply:
Yes, the thesis presents both the exact and an approximate operation (Proposition 3.1.30 and Proposition 3.1.31). We just need to find out which one is implemented here :detective:

Reply:
Apart from this question, I approve this PR :+1:

Reply:
the formula from the slides seems to me the same of proop 3.1.30, so I think this is the exact one?

Reply:
Agreed. I sent a suggestion to remove the other proposition.